### PR TITLE
[WIP] Better builds

### PR DESCRIPTION
**Pull request by naoisegolden**
WIP, do not merge.

This tries to improve the way Webpack builds de `dist` version.

TO DO:
* [ ] The generated `bundle.js` has absolute paths, and maybe ALL packages are only dev dependencies since all goes bundled into `bundle.js`. Perhaps check out a better dist version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,17 +13,18 @@
     "backbone.rel": "^0.2.4"
   },
   "devDependencies": {
+    "bootstrap": "^3.3.2",
```
Comment (enricostano) [inline package.json]: `bootstrap` shoud go under `dependencies`, right?

File: app/entry.js [hunk 2]
```diff
@@ -1 +1,9 @@
+// Javascript
 require("./js/app.js");
+
+// HTML
+require("./index.html");
```
Comment (enricostano) [inline app/entry.js]: mmmh interesting... :smiley:

File: package.json (hunk 1 above)
Comment (naoisegolden) [inline package.json]: you are right @enricostano

File: app/entry.js (hunk 2 above)
Comment (naoisegolden) [inline app/entry.js]: :wink:

**Comment by naoisegolden** [comment]
Updated info with a TODO task.

File: app/js/app.js [hunk 3]
```diff
@@ -1,8 +1,5 @@
 'use strict';
 
-require('../../lib/bootstrap-3.1.1-dist/css/bootstrap.css');
-require('../styles/main.css');
-require('imports?jQuery=jquery!../../lib/bootstrap-3.1.1-dist/js/bootstrap.js');
```
Comment (enricostano) [inline app/js/app.js]: Will this be replaced by the webpack/commonJS version of Bootstrap?

Comment (naoisegolden) [inline app/js/app.js]: Still trying to figure it out and make it work 😩

Comment (enricostano) [inline app/js/app.js]: push what you have here if you want, maybe I can help you!

**Pull request closed without merging**